src/main/java/io/sealights/plugins/sealightsjenkins/BeginAnalysis.java

```diff
@@ -819,29 +820,24 @@ private boolean validateAndTryUseToken(Logger logger, String token, SeaLightsPlu
         return true;
     }
 
-    private List<PomFile> getPomFiles(List<String> folders, String patterns, Logger logger, String pomPath) throws IOException, InterruptedException {
-        List<PomFile> pomFiles = new ArrayList<>();
-        boolean isParentPomInList = false;
```

Comment:
Why did you remove that flag?

Reply:
the point of this flag was to prevent putting duplicate `PomFile` of the root pom in the list. I took care of it differently (i.e. not putting the root pom in the list ever, and then creating the `PomFile` of the root in line 840.